AsyncDisplayKit/Layout/ASDimension.mm

```diff
@@ -66,12 +59,12 @@ @implementation NSNumber (ASDimension)
 
 - (ASDimension)as_pointDimension
 {
-  return ASDimensionMake(ASDimensionUnitPoints, self.floatValue);
+  return ASDimensionMake(ASDimensionUnitPoints, ASCGFloatFromNumber(self));
 }
 
 - (ASDimension)as_fractionDimension
```

Comment:
as_dimensionAsFraction?  Hmm this is a strange naming case, not sure it feels right currently...